src/jwplayer/players/MobileArticleVideoPlayer/MobileArticleVideoPlayer.tsx

```diff
@@ -75,27 +85,40 @@ export const MobileArticleVideoPlayerContent: React.FC<MobileArticleVideoPlayerP
 			recordVideoEvent(VIDEO_RECORD_EVENTS.JW_PLAYER_PLAYING_VIDEO);
 			return;
 		}
-	}, [onScreen, adComplete]);
+	}, [onScreen, adEngineComplete]);
 
 	return (
 		<>
 			<div ref={ref} className={clsx(styles.mobileArticleVideoTopPlaceholder, isScrollPlayer && `is-on-scroll-active`)}>
 				<MobileArticleVideoWrapper isScrollPlayer={isScrollPlayer}>
-					{adComplete && (
+					{jwpAdsSetupComplete.strategyRulesEnabled ? (
 						<>
-							<JwPlayerWrapper
+							<StrategyRulesWrapper
 								getDismissed={getDismissed}
 								config={getArticleVideoConfig(videoDetails)}
-								onReady={(playerInstance) => {
-									articlePlayerOnReady(videoDetails, playerInstance);
-									setIsPlayerReady(true);
-								}}
+								onReady={onPlayerInstanceReady}
 								stopAutoAdvanceOnExitViewport={false}
```

Comment:
I think it's not needed for StrategyRulesWrapper

Reply:
It was implemented this way on desktop so wanted to be consistent